spec/rollout_spec.rb

```diff
@@ -114,11 +114,11 @@
 
     context 'when feature is NOT rolled out for the user' do
       subject { rollout_except(feature, user) }
-      it { is_expected.not_to be_active(feature, user) }
+      it { is_expected.to be_active(feature, user) }
```

Comment:
Wow, really?! If a feature is not rolled out for the user, then that feature should be active for that user? That sounds strange. I would write a comment here to describe why the behavior of the system is the opposite of what I would expect from just reading the names of things. :) Legacy systems can be strange.

Reply:
ah... I've been checking how failing tests reads and committed with it 🤦‍♂ 
sorry for confusion, fixed!